hpat/tests/test_series.py

```diff
@@ -2612,5 +2612,83 @@ def test_series_count_impl(S):
             result = hpat_func(S)
             self.assertEqual(result, result_ref)
 
+
+    def test_series_pct_change(self):
+        def test_series_pct_change_impl(S, periods, method):
+            return S.pct_change(periods, method)
+
+        hpat_func = hpat.jit(test_series_pct_change_impl)
+        test_input_data = [
+            [],
+            [np.nan, np.nan, np.nan],
+            [np.nan, np.nan, np.inf],
+            [0] * 8,
+            [0, 0, 0, np.nan, np.nan, 0, 0, np.nan, np.inf, 0, 0, np.inf, np.inf],
+            [1.1, 0.3, np.nan, 1, np.inf, 0, 1.1, np.nan, 2.2, np.inf, 2, 2],
+            [1, 2, 3, 4, np.nan, np.inf, 0, 0, np.nan, np.nan]
+        ]
+        for input_data in test_input_data:
+            S = pd.Series(input_data)
+            for periods in [0, 1, 2, 5, 10, -1, -2, -5]:
+                for method in [None, "pad", "ffill", "backfill", "bfill"]:
+                    result_ref = test_series_pct_change_impl(S, periods, method)
+                    result = hpat_func(S, periods, method)
+                    pd.testing.assert_series_equal(result, result_ref)
+
+    def test_series_pct_change_str(self):
+        def test_series_pct_change_impl(S):
+            return S.pct_change()
+
+        hpat_func = hpat.jit(test_series_pct_change_impl)
+        S = pd.Series(["abababa", "aaaaa", "bbbbb"])
+        with self.assertRaises(AssertionError):
```

Comment:
Did you run tests before pushing the commit? The tests really catch `AssertionError`?
Could you please use `assertIn()` to check exception message?